BraveShareTo/ShareToBraveViewController.swift

```diff
@@ -33,19 +34,22 @@ class ShareToBraveViewController: SLComposeServiceViewController {
         }
         
         provider.loadItem(of: provider.isUrl ? kUTTypeURL : kUTTypeText) { item, error in
-            var urlItem: URL!
+            var urlItem: URL?
             
             // We can get urls from other apps as a kUTTypeText type, for example from Apple's mail.app.
             if let text = item as? String {
-                urlItem = URL(string: text)
+                urlItem = text.firstURL
             } else if let url = item as? URL {
-                urlItem = url
+                urlItem = url.absoluteString.firstURL
```

Comment:
Is this necessary? If the item is a URL, it by nature, is a single URL, right?

Reply:
It's not super necessary, more like a just-in-case to keep logic consistent